### PR TITLE
Replace athlete name with the symbol on the athlete page dialogs

### PR DESCRIPTION
**Pull request by Friendlygeckos**
# Description
replace athlete names with the symbol

Fixes Jira Ticket # 462
https://athletex.atlassian.net/browse/AX-462

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Is this a UI Change? If so please include screenshot of before and after states below
## Before
image
image
## After
image
image

# How Has This Been Tested?
Ran the app on Chrome using web-server

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/dialogs/sell/SellDialog.dart [hunk 1]
```diff
@@ -334,10 +341,11 @@ class _SellDialogState extends State<SellDialog> {
                               ),
                             ),
                             Container(width: 15),
-                            Expanded(
+                            Expanded(                         
                               child: Text(
-                                widget.athleteName + " " + _currentTokenTypeSelection.name + " " +
-                                    " APT",
+                                _currentTokenTypeSelection == TokenType.Long ? 
+                                getLongAthleteSymbol(widget.athleteId) + " APT" :
```
Comment (MichaelFrank0317) [inline lib/dialogs/sell/SellDialog.dart:347]: You could use $ rather then + operator

File: lib/dialogs/sell/SellDialog.dart [hunk 2]
```diff
@@ -334,10 +341,11 @@ class _SellDialogState extends State<SellDialog> {
...
+                                getShortAthleteSymbol(widget.athleteId) + " APT",
```
Comment (MichaelFrank0317) [inline lib/dialogs/sell/SellDialog.dart:348]: you could use $ instead of + operator

**Review by MichaelFrank0317 — CHANGES_REQUESTED** [review]

File: lib/dialogs/sell/SellDialog.dart (hunk 2 above)
Comment (Friendlygeckos) [reply lib/dialogs/sell/SellDialog.dart]: fixed

**Review by MichaelFrank0317 — APPROVED** [review]

File: pubspec.lock [hunk 3]
```diff
@@ -203,7 +203,7 @@ packages:
       name: collection
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.16.0"
+    version: "1.15.0"
```
Comment (mretana1999) [inline pubspec.lock:206]: can you undo these changes to pubspec.lock?

File: pubspec.lock [hunk 4]
```diff
@@ -1277,5 +1277,5 @@ packages:
     source: hosted
     version: "3.1.0"
 sdks:
-  dart: ">=2.17.0-0 <3.0.0"
+  dart: ">=2.16.0 <3.0.0"
```
Comment (mretana1999) [inline pubspec.lock:1280]: same as comment above

**Review by mretana1999 — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 3 above)
Comment (Friendlygeckos) [reply pubspec.lock]: done

File: pubspec.lock (hunk 4 above)
Comment (Friendlygeckos) [reply pubspec.lock]: done

**Review by mretana1999 — APPROVED** [review]
